apkid/rules/apk/packers.yara

```diff
@@ -1176,3 +1176,20 @@ rule manxi_sec : packer
   condition:
     is_apk and any of them
 }
+
+rule dexprotectx : packer
+{
+  meta:
+    description = "DexProtect X"
+    url         = "https://dexprotectx.pro"
```

Comment:
Could we also include an extra `url` tag with the Telegram channel link?

`url2   = https://t.me/DexShell_x`